### ISSUE TITLE
Is it planned to support Cakephp-4?

### ISSUE THREAD

**Issue by davalb** [Oefenweb/cakephp-vat-number-check#18]
Do you plan to update this plugin or do you know of a fork that supports cakephp-4? Thanks!

**Comment by tersmitten** [comment]
Yes, but the application we're using this plugin in is still CakePHP 3 and not soon the be updated to 4, so it may take a while.
